src/lib/Preview.js

```diff
@@ -213,6 +222,44 @@ class Preview extends EventEmitter {
         this.file = undefined;
     }
 
+    /**
+     * Reloads the current preview. Cleans up existing preview and re-loads from cache.
+     * Note that reload() will not do anything if either:
+     *   - skipServerUpdate is true (either passed in or defined in preview options) AND cached file is not valid
+     *   - skipServerUpdate is false AND there is no cached file ID
+     *
+     * @public
+     * @param {boolean} skipServerUpdate - Whether or not to update file info from server
+     * @return {void}
+     */
+    reload(skipServerUpdate) {
+        // If not passed in, default to Preview option for skipping server update
+        if (typeof skipServerUpdate === 'undefined') {
```

Comment:
If someone passes in `null`, `this.options.skipServerUpdate` won't be respected

Reply:
We could check if it's not a boolean

Reply:
I think that's fine, you'd have to explicitly pass in `null` and in that case, it evaluates to falsy.